code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -473,16 +488,18 @@ There are several things that need to be remembered:
 
 		var/handled_by_bodytype = TRUE
 
+		var/species_override_icon // SKYRAT EDIT ADDITION
+
 		//More currently unused digitigrade handling
 		if(dna.species.bodytype & BODYTYPE_DIGITIGRADE)
 			if(worn_item.supports_variations_flags & CLOTHING_DIGITIGRADE_VARIATION)
-				icon_file = DIGITIGRADE_SUIT_FILE
+				species_override_icon = wear_suit.worn_icon_digi || DIGITIGRADE_SUIT_FILE // SKYRAT EDIT CHANGE
```

Comment:
why cant this just use `icon_file`
at worst use
suit_overlay = wear_suit.build_worn_icon(default_layer = SUIT_LAYER, default_icon_file = icon_file, override_file = handled_by_bodytype ? override_file : null)

Reply:
I guess it can